.travis.yml

```diff
@@ -1,4 +1,6 @@
 # Documentation: http://docs.travis-ci.com/user/languages/julia/
+# for some reason, Linux builds are failing when we are at 50 commits, https://github.com/travis-ci/travis-ci/issues/4942
+# adding these lines to get commit 51
```

Comment:
It is very strange. I'd remove this comment now as it shouldn't be an issue any more.